packages/api-gateway/src/drivers/http/adapters/geolocation/handlers.js

```diff
@@ -47,7 +47,28 @@ async function getAddress(req, reply) {
   }
 }
 
+async function deleteGeolocationPlace(req, reply) {
+  try {
+    const { placeId } = req.query;
+
+    req.log.info('[http-server]: Deleting place in Geolocation database: ', {
+      placeId,
+    });
+
+    // const address = await this.geocolocationServices.deleteGeolocationPlace(placeId);
```

Comment:
@CETHER what do you think if we remove this dead code?

Reply:
Yes! We found another way to avoid this line and mock the function